modules/core/src/norm.simd.hpp

```diff
@@ -383,8 +381,7 @@ struct NormDiffInf_SIMD<uchar, int> {
         }
         s = (int)v_reduce_max(v_max(v_max(v_max(r0, r1), r2), r3));
         for (; j < n; j++) {
-            int v = src1[j] - src2[j];
-            s = std::max(s, (int)cv_abs(v));
```

Comment:
cv_abs is "public" function and defined in modules/core/include/opencv2/core/base.hpp. It should be fixed too, in case of overflow. Why do you drop it here?

Reply:
Just to keep in sync with the masked one to avoid any confusion. See line 1261 below.

https://github.com/opencv/opencv/blob/7d87f3cda6837f6ae1f8346d9098c06cc081de88/modules/core/src/norm.simd.hpp#L1251-L1268

I'm okay with either cv_abs or std::abs solution.